controllers/tempo/sa_filter.go

```diff
@@ -0,0 +1,52 @@
+package controllers
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/grafana/tempo-operator/internal/manifests/manifestutils"
+)
+
+// Filter service account objects already created and modified by OCP. e.g  bound service account tokens
```

Comment:
```suggestion
// Filter service account objects already created and modified by other controller. e.g  bound service account tokens
```
I would say this because we are not checking for any specific OpenShift feature